connector/routingconnector/logs_test.go

```diff
@@ -434,6 +434,9 @@ func TestLogsConnectorDetailed(t *testing.T) {
 	isLogX := `body == "logX"`
 	isLogY := `body == "logY"`
 
+	isBodyString := `IsString(body) == true`
+	isBodyMap := `IsMap(body) == true`
```

Comment:
Are there any other tests we should add from the StandardConverters?

Reply:
I think with the change of scope these should be good as a "candidate converter function" inclusion tests but let me update test terminology to probably better indicate just testing the inclusion within this specific effort.